pages/all-about.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>All About DIK-DIKS!</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/all-about.css">
+  <link href="https://fonts.googleapis.com/css2?family=Gloria+Hallelujah&display=swap" rel="stylesheet">
+
+  <style>
+  .dikdik-banner-container {
+      position: relative;
+      z-index: 1000;
+      width: 100%;
+      background-color: #f5e9d2; 
+  }
+  </style>
+
+</head>
+
+<body>
+  <div class="page-wrapper">
+    <header>
+      <h1>All About Dik-Diks!</h1>
+      <nav>
+        <ul>
+          <li><a href="../index.html">Home</a></li>
+          <li><a href="all-about.html">All About Dik-Diks</a></li>
+          <li><a href="fun-facts.html">Fun Facts</a></li>
+          <li><a href="gallery.html">Gallery</a></li>
+        </ul>
+      </nav>
+    </header>
+
+  <div class="dikdik-banner-container"> 
+    <div class="dikdik-banner">
+        <img src="../assets/dikdik-stare.jpg" alt="Close-up of a dik-dik in natural habitat">
+    </div>
+  </div>
+
+  <div class="facts-heading">
+  <h2>Interesting Facts About Dik-Diks:</h2>
+  </div>
```

Comment:
If I'm interpreting this correctly, the `header` tag is a sibling with the `div` tag that has the class "dikdik-banner-container".

Therefore, the div tag needs to be indented one a level deeper. 

Then the `div` tag with class "facts-heading" also needs to be indented one level deeper. 

```suggestion
    <header>
      <h1>All About Dik-Diks!</h1>
      <nav>
        <ul>
          <li><a href="../index.html">Home</a></li>
          <li><a href="all-about.html">All About Dik-Diks</a></li>
          <li><a href="fun-facts.html">Fun Facts</a></li>
          <li><a href="gallery.html">Gallery</a></li>
        </ul>
      </nav>
    </header>

    <div class="dikdik-banner-container"> 
      <div class="dikdik-banner">
        <img src="../assets/dikdik-stare.jpg" alt="Close-up of a dik-dik in natural habitat">
      </div>
    </div>

    <div class="facts-heading">
      <h2>Interesting Facts About Dik-Diks:</h2>
    </div>
```

You can use an online HTML formatter to help you fix up indentation issues or you can use VSCode (cmd shift p will open up a command palette --> Format Document)